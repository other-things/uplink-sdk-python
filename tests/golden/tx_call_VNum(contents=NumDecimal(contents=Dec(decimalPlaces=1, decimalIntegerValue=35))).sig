r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 85275480492201098372709752916029628493520843830266699616351958711403283996330
