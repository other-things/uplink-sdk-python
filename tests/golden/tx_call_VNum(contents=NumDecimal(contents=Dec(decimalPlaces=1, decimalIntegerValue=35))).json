{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 35,
                                "decimalPlaces": 1,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE04NTI3NTQ4MDQ5MjIwMTA5ODM3MjcwOTc1MjkxNjAyOTYyODQ5MzUyMDg0MzgzMDI2NjY5OTYxNjM1MTk1ODcxMTQwMzI4Mzk5NjMzMA=="
}